### PR TITLE
Improvement/new env vars

### PR DESCRIPTION
**Pull request by BondarenkoStas**
FCI_REPO_SLUG, FCI_PULL_REQUEST, FCI_PULL_REQUEST_NUMBER env vars are already in live added by me.
FCI_CLONE_DEPTH is already in live added by Armold.

FCI_BRANCH shows pr's source branch (not destination)

### CODE REVIEW

**Comment by VeArnold** [comment]
suggestion to also add PROJECT_BUILD_INDEX - which is the much awaited global build index, that takes account all of the workflows (it was just deployed)

File: content/building/environment-variables.md [hunk 1]
```diff
@@ -23,11 +23,16 @@ Here you'll find some of the read-only environment variables explained.
...
 | CONTINUOUS_INTEGRATION   | true                                                                                                            |
-| BUILD_NUMBER             | Number of the build for this project in Codemagic                                                               |
+| BUILD_NUMBER             | Number of the build for this project in Codemagic for the given workflow                                        |
+| PROJECT_BUILD_NUMBER     | Number of the build for this project in Codemagic                                                               |
 | FLUTTER_ROOT             | Absolute path to Flutter SDK                                                                                    |
-| FCI_BRANCH               | The current branch being built, for pull requests it is the destination branch                                  |
+| FCI_BRANCH               | The current branch being built, for pull requests it is the source branch                                       |
+| FCI_REPO_SLUG            | The slug of the repository that is currently being built in the form owner_name/repository_name. Unset in case of custom source added repository|
```
Comment (helinanever) [inline content/building/environment-variables.md:30]: * format `owner_name/repository_name` with upticks
* Fix grammar: "Unset for repositories added from custom source"

File: content/building/environment-variables.md [hunk 2]
```diff
@@ -23,11 +23,16 @@ Here you'll find some of the read-only environment variables explained.
...
 | FCI_COMMIT               | Commit hash that is currently being built by Codemagic, for pull request builds it is the hash of the source commit |
-| FCI_CLONE_UNSHALLOW      | If set to `true`, performs a full clone of the repository instead of top 50 commits                              |
+| FCI_PULL_REQUEST         | `true`, if the current build is building a pull request, `false` otherwise                                      |
```
Comment (helinanever) [inline content/building/environment-variables.md:32]: Remove comma after `true`

File: content/building/environment-variables.md [hunk 3]
```diff
@@ -23,11 +23,16 @@ Here you'll find some of the read-only environment variables explained.
...
+| FCI_PULL_REQUEST_NUMBER  | Set to Integer ID of the pull request for the Git provider (Bitbucket, Github etc) if the current build is building a pull request, unset otherwise|
+| FCI_CLONE_UNSHALLOW      | If set to `true`, performs a full clone of the repository instead of top 50 commits                             |
+| FCI_CLONE_DEPTH          | Specifies the clone depth which allows to decrease default fetching time when is set lower than 50              |
```
Comment (helinanever) [inline content/building/environment-variables.md:35]: Specifies the number of commits to be fetched from the repository when cloning, default: 50. Specifying a smaller number can decrease the default fetching time.

**Review by helinanever — APPROVED** [review]
LGTM

**Pull request merged**